### ISSUE TITLE
chat background image removal unability

### ISSUE THREAD

**Issue by ghost** [telegramdesktop/tdesktop#4515]
There is no option that deletes the chat background image and turn back to the default one once set.

**Comment by stek29** [comment]
What for? Can't you just pick it from gallery?

**Comment by gh00sty** [comment]
On dark mode theme 
@stek29 what you refer to is setting another background image
what i refer to is going back to default dark mode theme background settings which is only achievable by removing the background which is not possible i guess

**Comment by vlad1777d** [comment]
Maybe there are some workarounds.
For example, to remove it from theme text file...

**Comment by rpodric** [comment]
I found today, using v1.37 on Windows, that it is possible to revert: by going back to the default Daisies background and then toggling night mode on/off. Once you do that, you'll get back to night mode with no background.

The need to be on Daisies would seem like a bug to me. I think it'll tend to come up for people once they try out night mode and then decide to play with some backgrounds. They might think, like I did, that backgrounds and night mode are not the best pairing, and so they'll want to revert...but they can't without knowing this non-intuitive trick. The only reason I thought to try it is I began to wonder how I got dark mode with no background in the first place.

There is a "blank" box at the bottom of the chat background gallery that I thought might do it, but it doesn't.
